engine/pom.xml

```diff
@@ -23,7 +23,7 @@
     <distributionManagement>
         <site>
             <id>site</id>
-            <url>http://validator.hibernate.org</url>
+            <url>http://hibernate.org/validator</url>
```

Comment:
>  Last but not least, there is search.hibernate.org and that's what's used in the Search POM.

Sure that's not the best choice either. But here I was just scanning the whole project for links, I wouldn't create a JIRA and a PR just to fix a single link ;)

> I am not sure about "canonical representation". Just because one URL is a redirect, does not mean it is less important than the URL it redirects to. 

It is in context of SEO, "canonical" is a technical term of the field. This is de-valuing the SEO value for your project page, you need to make it clear across all your content which one is the "right" way to spell your homepage.

But +1 to spend cycles on more valuable stuff ;) I was just reviewing all links and went overboard with perfectionism.